app/presenters/queries/content_item_presenter.rb

```diff
@@ -125,6 +128,19 @@ def publication_state_sql
         SQL
       end
 
+      def state_history_sql
+        <<-SQL
+          (
+            SELECT json_agg((number, name))
+            FROM content_items c
```

Comment:
I am not sure about the aliasing of the table names, I find that I have to keep doing the conversion from `c` to `content_items` in my head when reading it.